### DISCUSSION TITLE
Renaming engines

### DISCUSSION THREAD

**Discussion by ZachZent** [tsunamayo/Starship-EVO#4242] [Suggestions Done]
It would be easier for new players if the "block" thruster was renamed the cruise engine rather than thruster on its own and the "thruster small" renamed to Thruster (round/square)

![image](https://user-images.githubusercontent.com/36980446/138789630-56f51e8d-c280-412b-8b53-5632984644ae.png)
![image](https://user-images.githubusercontent.com/36980446/138789649-3bb91461-789b-448a-98b3-75d391c21695.png)

**Comment by tsunamayo** [comment]
@ZachZent Yeah you are 100 right...

**Comment by tsunamayo** [comment]
Done for 22w07b. Cheers
